.github/workflows/build.yml

```diff
@@ -44,3 +44,15 @@ jobs:
           key: node_modules-${{ hashFiles('yarn.lock') }}
       - run: yarn
       - run: yarn lint
+  chromatic-deployment:
```

Comment:
We shouldn't be running this on the pull_request event but only on push event as recommend by the docs see https://www.chromatic.com/docs/github-actions#recommended-configuration-for-build-events